docs/xml/enumerations/vote_variation.rst

```diff
@@ -1,35 +1,32 @@
 VoteVariation
 =============
 
-.. todo::
-
-   Document VoteVariation
-
+    
 +----------------------+----------------------------------------------------------------------------------+
 | Name                 | Description                                                                      |
 |                      |                                                                                  |
 +======================+==================================================================================+
-| 1-of-m               |                                                                                  |
-+----------------------+----------------------------------------------------------------------------------+
-| approval             |                                                                                  |
+| 1-of-m               | When a voter can select only one candidate in a contest.                         |
 +----------------------+----------------------------------------------------------------------------------+
-| cumulative           |                                                                                  |
+| approval             | When a voter can select as many candidates as desired.                           |
```

Comment:
If you want to include the description, then I would say:

    Approval voting, a system that allows voters to select as many candidates as desired.